sdk-extensions/autoconfigure/src/testOtlp/java/io/opentelemetry/sdk/autoconfigure/OtlpGrpcServerExtension.java

```diff
@@ -164,8 +165,10 @@ static MetricData generateFakeMetric() {
   }
 
   static LogData generateFakeLog() {
-    return LogDataBuilder.create(Resource.empty(), InstrumentationScopeInfo.empty())
-        .setEpoch(Instant.now())
+    return TestLogData.builder()
+        .setResource(Resource.empty())
+        .setInstrumentationScopeInfo(InstrumentationScopeInfo.empty())
+        .setEpoch(Instant.now().toEpochMilli(), TimeUnit.MILLISECONDS)
```

Comment:
curious why not just use `Instant.now()` here still since it looks like that's on the builder. (also applies to a bunch of other test classes)

Reply:
Ah the `setEpoch(Instant)` method on the builder was a late addition and I must have forgotten to undo some code in some places. Fixed!